lib/templates/mixins/seo/getMeta.js

```diff
@@ -38,15 +38,36 @@ export default {
         })
       }
 
+      meta.link = [];
+      if (this.page.languages.filter(item => item.available == 1).length > 1) {
```

Comment:
To avoid two loops I recommend just check `this.page.languages.length > 1` and after that we can filter and add available languages to head:

```js
this.page.languages.forEach(item => {
          if (item.available) {
            meta.link.push({
              hid: item.title,
              rel: 'alternate',
              hreflang: item.hreflang,
              href: baseUrl + item.link
            })
          }
        })
```